src/cloudWatchLogs/registry/logStreamRegistry.ts

```diff
@@ -172,26 +207,26 @@ export class CloudWatchLogStreamData {
     data: CloudWatchLogs.OutputLogEvents = []
     next?: {
         token: CloudWatchLogs.NextToken
-        expiry: Date
     }
     previous?: {
         token: CloudWatchLogs.NextToken
-        expiry: Date
     }
-    mostRecentExpiry?: Date
+    busy?: boolean
```

Comment:
nit/curious: why is `busy` optional?

Reply:
Dumb reason, since this is a class instead of an interface it would have to be initialized. I can initialize to false if you'd rather (or remove it altogether, the functionality that this backs doesn't work and I'm unsure if it ever will (adjusting the codelens to show "Loading" after clicking it while waiting for results)).

Reply:
i would init to false, is there any reason not to do that in general? Not a blocker anyway.